matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -685,7 +550,10 @@ internal class DefaultCryptoService @Inject constructor(
      */
     @Throws(MXCryptoError::class)
     override fun decryptEvent(event: Event, timeline: String): MXEventDecryptionResult {
-        return internalDecryptEvent(event, timeline)
+
+        return runBlocking {
+            olmMachine!!.decryptRoomEvent(event)
```

Comment:
the `timeline: String` parameter is not used here?

Reply:
Well it's not really clear what this is supposed to be used for, I don't think I saw any docs for this, the rust-sdk doesn't really know what a timeline is, all the info necessary to decrypt a megolm encrypted event is part of the event.

Reply:
The javadoc says `It is used to prevent replay attack.`, I have no other info on it

Reply:
That's handled inside the rust-sdk, all the info to combat replay attacks is contained in the event as well.

Reply:
ok, so we will do the cleanup later